display/release/BigMemory-Max-4.3.md

```diff
@@ -15,10 +15,11 @@ Fixes are cumulative from version to version.
 
 ### **Deprecation Notices**
 
-Software AG intends to deprecate and remove from distribution the WAN Replication Service/Module in the next release (4.3.10, late 2021).
+*Announcement March 2021: Software AG intends to deprecate and remove from distribution the WAN Replication Service/Module in the next release (4.3.10, late 2021).*
```

Comment:
May not work without a space after the star

Reply:
hmm. in my md viewier it doesn't work if there *is* a space